### PR TITLE
Shifting Magento tab into configuration tab #7837

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

**Comment by vishu1993** [comment]
channel2

File: view/sale_channel_form.xml [hunk 1]
```diff
@@ -1,66 +1,67 @@
...
+            <field name="magento_order_prefix"/>
+            <field name="magento_order_states" colspan="8"/>
+        </page>
+        <page string="Carrier / Shipping Methods" id="carriers" states="{'invisible': Not(Eval('source') == 'magento')}">
+            <field name="magento_carriers" colspan="4"/>
+        </page>
+        <page string="Price Tiers" id="price_tiers" states="{'invisible': Not(Eval('source') == 'magento')}">
+            <field name="magento_price_tiers" colspan="8"/>
+        </page>
+        <page string="Import / Export Time" id="import_export_time" states="{'invisible': Not(Eval('source') == 'magento')}">
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: I think it should be common tab in sale channel ( inside configuration )

File: view/sale_channel_form.xml [hunk 2]
```diff
@@ -1,66 +1,67 @@
...
+            <label name="magento_last_order_import_time"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: Import / Export is new tab in sale channel now. Please reuse that

File: view/sale_channel_form.xml [hunk 3]
```diff
@@ -1,66 +1,70 @@
...
     </xpath>
-    <xpath expr="/form[@string='Sale Channel']" position="inside">
+    <xpath expr="/form/notebook/page[@id='configuration']" position="inside">
         <group id="magento_buttons" col="3" colspan="6" states="{'invisible': Not(Eval('source') == 'magento')}">
             <button string="Configure Magento Connection" name="configure_magento_connection"/>
             <button string="Import Magento Order States" name="import_magento_order_states"/>
             <button string="Import Magento Carriers" name="import_magento_carriers"/>
         </group>
-    </xpath>
+    </xpath>
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: remove trailing space

**Pull request merged**